packages/react/src/avatar/avatar.tsx

```diff
@@ -32,14 +34,26 @@ const safeText = (text: string): string => {
 
 export const Avatar = React.forwardRef(
   (props: AvatarProps, ref: ReactRef<HTMLSpanElement>) => {
-    const { src, text, icon, alt, className, ...otherProps } = props;
+    const {
+      src,
+      text,
+      icon,
+      alt,
+      className,
+      imgRef: imgRefProp,
+      ...otherProps
+    } = props;
 
     const domRef = useDOMRef(ref);
 
     const showText = !src;
     const [ready, setReady] = useState(false);
 
     const imgRef = useRef<HTMLImageElement>(null);
+    const handleImgRef = useForkRef<HTMLImageElement, HTMLImageElement>(
```

Comment:
There is a function for this already in the utils folder called `useDomRef`. I would use that instead.